test/simulator/main/org/apache/cassandra/simulator/paxos/PairOfSequencesAccordSimulation.java

```diff
@@ -182,7 +191,7 @@ public class ReadWriteOperation extends Operation
 
         public ReadWriteOperation(int id, int[] primaryKeys, IntHashSet reads, IntHashSet writes, IInvokableInstance instance)
         {
-            super(primaryKeys, id, instance, "Accord", createQuery(id, reads, writes));
+            super(primaryKeys, id, instance, "Accord ReadWrite Txn", createQuery(id, reads, writes));
```

Comment:
while running the tests in the loop I wanted to validate what actions were run and used `Action.description()` to get this visibility.  Just saying `Accord` felt weird, so made a bit more verbose.

This change itself is a no-op in the single partition case, there was a bug there that caused the `idString` to get dropped